examples/image-carousel/index.js

```diff
@@ -0,0 +1,44 @@
+const Tram = require('../../tram-one')
+const app = new Tram()
+
+const Image = require('./elements/Image')
+const ImageDeck = require('./elements/ImageDeck')
+
+const ImageCarouselActions = require('./reducers/ImageCarouselActions')
+
+const html = Tram.html({
+  Image,
+  ImageDeck
+})
+
+const textCenter = `
+  text-align : center;
+`
+
+const home = (store, actions) => {
+
+  const onSelectImage = (index) => () => {
+    actions.onSelectImage(index)
+  }
+
+  const onPreviousSelect = () => {
+    actions.onPreviousSelect();
+  }
+
+  const onNextImage = () => {
+    actions.onNextImage();
+  }
+
+  return html`
+    <div>
+      <h1 style=${textCenter}> Tram-One Image Carousel 🚋 </h1>
+      <Image tiles=${store.data.tiles} selected=${store.data.selected} onNextImage=${onNextImage} onPreviousSelect=${onPreviousSelect}></Image>
```

Comment:
Assuming you have Tram v3 (just updated today), you can actually self close this :D 
So you can just do 
```js
<Image tiles=${store.data.tiles} selected=${store.data.selected} onNextImage=${onNextImage} onPreviousSelect=${onPreviousSelect} />
```

Reply:
Same with `ImageDeck` below